src/components/AppContainer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component, Fragment } from "react";
+import HeaderApp from "./Header";
+import FooterApp from "./Footer";
+import "../styles/AppContainer.css";
+
+export default class AppContainer extends Component {
+  render() {
+    return (
+      <Fragment>
```

Comment:
As there is already wrapped div inside and it is the only child this Fragment is excess - please remove it.